tests/generation/FileGlobs.py

```diff
@@ -30,11 +31,8 @@ def test_get_project_files(self):
         open(os.path.join("ignore_dir", "src.js"), "w").close()
 
         with suppress_stdout(), simulate_console_inputs("ignore_dir/**"):
-            res, _ = get_project_files(self.log_printer, self.printer, os.getcwd())
-            self.assertIn(os.path.join(os.getcwd(), "src", "file.c"), res)
-            self.assertIn(os.path.join(os.getcwd(), "root.c"), res)
```

Comment:
Why did you remove these tests? These should still work when you give `noninteractive` as `False` to `get_project_files`, right?